README.md

```diff
@@ -118,13 +118,16 @@ $ cd terraform
 $ cp example.tfvars terraform.tfvars
 ```
 
-Fill in your username as the `cert_owner`:
+Fill in your username as the `cert_owner` via:
 
 ```sh
 $ sed -i "s/FIXME_USER/$USER/" terraform.tfvars
 ```
 
-Check terraform.tfvars and change any values you would like to change. Note that the values given in examples.tfvars is NOT completely AWS free tier eligible, as they include t2.small and t2.medium instances. We do not recommend using t2.micro instances, as they were unable to compile solidity during testing.
+Check terraform.tfvars and change any values you would like to change:
+- You may need to modify the network ID to one which is not in use.  
```

Comment:
While you're doing this, explicitly mention that we expect naming conflicts running two networks with the same network id in the same AWS account

Reply:
Didn't see this comment, gonna cut one more commit to fix it

Reply:
Most recent commit addresses both this & the other one -- should be good to go!